README.md

```diff
@@ -126,6 +140,13 @@ Write code to do the following:
 Write your code here:
 ```ruby
 # code here
+genie = Genie.create(name: "Genie")
+lamp = Lamp.create(wishes_remaining: 3)
+genie.lamp_id = lamp.id
+lamp.update(wishes_remaining: 1)
+jafar = Genie.create(name: "Jafar")
+jafar.lamps.create(wishes_remaining: 3)
+genie.update(wishes_remaining: nil)
```

Comment:
This doesn't work as we expect... genies don't have a `wishes_remaining` property, only lamps do.

We're looking for `genie.lamp = nil`.